pkgs/tools/security/dkimpy/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, pythonPackages, fetchurl, openssl, makeWrapper }:
```

Comment:
`callPackage from` from `python-packages.nix` to an expression outside `/development/python-modules` is wrong. If this is an application then it should be called straight from `all-packages.nix`.